main.js

```diff
@@ -49,6 +49,7 @@ module.exports = function(options) {
 		withAnonMiddleware: false,
 		withBackendAuthentication: false,
 		// TODO always default to false for next major version
```

Comment:
could you move this comment down a line as it should be with the withAssets option